test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -73,29 +54,41 @@ public class Steps : IDisposable
         private HttpContent _postContent;
         private BearerToken _token;
         public string RequestIdKey = "OcRequestId";
-        private readonly Random _random;
+        private readonly Random _random = new();
+        private readonly string _ocelotConfigFileName = $"{Guid.NewGuid():N}-ocelot.json";
```

Comment:
I can accept this: `private readonly Random _random = new();`
But I cannot accept on line 58. It contains business logic which is hidden in definition of the `_ocelotConfigFileName` member.
So, it is better to move all initializations to constructor.

Reply:
well...